main.js

```diff
@@ -74,11 +79,11 @@ function subOneToPurchaseChoice (itemIndex) {
         if (i === itemIndex) {
             if (purchaseItems[i].numSelected !== 0) {
                 purchaseItems[i].numSelected -= 1;
+                changeTotal (itemIndex);
             }
         }
     }
     updatePurchaseMenu ();
-    changeTotal (itemIndex);
     console.log('subOneToPurchaseChoice');
 }
 
```

Comment:
Having to add 'add' to the argument for the changeTotal function could be confusing. Is there a better way to do this that could utilize a negative number?